bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "origin-web-catalog",
-  "version": "0.0.34",
+  "version": "0.0.35",
```

Comment:
We  shouldn't bump the version as part of a separate PR. Bumping the version should be its own PR to avoid conflicts and version churn.